plugins/maxhitplugin

```diff
@@ -1,2 +1,2 @@
 repository=https://github.com/Matt12yearslater/max-hit.git
-commit=23d328cb160dd471431fd88fbb8268b8f01f01a2
+commit=5edb716712f06a8dd3de933d414f7629ff19a24c
```

Comment:
```suggestion
commit=5edb716712f06a8dd3de933d414f7629ff19a24c
authors=Matt12yearslater,Seebity
```

@Matt12yearslater confirming you are okay with Seebity being able to update the commit hash?